src/client.rs

```diff
@@ -26,6 +27,12 @@ impl HttpsConnector<HttpConnector> {
     ///
     /// This uses hyper's default `HttpConnector`, and default `TlsConnector`.
     /// If you wish to use something besides the defaults, use `From::from`.
+    ///
+    /// # Warning
```

Comment:
I'd change this to "Note" instead of "Warning".

Reply:
I originally had it as Note, i'll change it back.